src/pull-request/pull-request-diff.html

```diff
@@ -69,6 +79,10 @@
     Polymer({
       is: 'pull-request-diff',
       properties: {
+        editable: {
+          type: Boolean,
+          value: false
```

Comment:
`<pull-request-chunk>` have by default `editable = false`. So this should not be necessary.